backend/scripts/load_h5ad.py

```diff
@@ -0,0 +1,223 @@
+organism = adata.uns["organism"]
+organism_ontology = adata.uns["organism_ontology"]
+tissue = adata.obs["tissue"].dtype.categories[0]
+tissue_ontology = adata.obs["tissue_ontology"].dtype.categories[0]
+assay = adata.obs["assay"].dtype.categories[0]
+assay_ontology = adata.obs["assay_ontology"].dtype.categories[0]
+disease = adata.obs["disease"].dtype.categories[0]
+disease_ontology = adata.obs["disease_ontology"].dtype.categories[0]
+sex = "NA"  # Not available. Required?
+ethnicity = "NA"  # Not available. Required?
+ethnicity_ontology = "NA"  # Not available. Required?
+source_data_location = "NA"  # Not available. Required?
+preprint_doi = adata.uns["preprint_doi"]
+publication_doi = "NA"  # Not available. Required?
+
+dataset = DbDataset(
+    id=dataset_id,
+    revision=revision,
+    name=dataset_name,
+    organism=organism,
+    organism_ontology=organism_ontology,
+    tissue=tissue,
+    tissue_ontology=tissue_ontology,
+    assay=assay,
+    assay_ontology=assay_ontology,
+    disease=disease,
+    disease_ontology=disease_ontology,
+    sex=sex,
+    ethnicity=ethnicity,
+    ethnicity_ontology=ethnicity_ontology,
+    source_data_location=source_data_location,
+    preprint_doi=preprint_doi,
+    publication_doi=publication_doi,
+)
+print(dataset)
+session.add(dataset)
+session.commit()
+
+# project dataset
+pd_id = str(uuid.uuid4())
+pd_project_id = project_id
+pd_project_status = ProjectStatus.LIVE.name
+pd_dataset_id = dataset_id
+
+project_dataset = DbProjectDataset(
+    id=pd_id, project_id=pd_project_id, project_status=pd_project_status, dataset_id=pd_dataset_id
+)
+print(project_dataset)
+session.add(project_dataset)
+session.commit()
+
+# project links
+# RAW DATA
+project_links = []
+for link in adata.uns["project_raw_data_links"]:
+    project_link_id = str(uuid.uuid4())
+    pl_p_id = project_id
+    pl_p_status = ProjectStatus.LIVE.name
+    link_url = link
+    link_type = ProjectLinkType.RAW_DATA.name
+
+    project_link = DbProjectLink(
+        id=project_link_id, project_id=pl_p_id, project_status=pl_p_status, link_url=link_url, link_type=link_type
+    )
+    print(project_link)
+    project_links.append(project_link)
+# OTHER
+for link in adata.uns["project_other_links"]:
+    project_link_id = str(uuid.uuid4())
+    pl_p_id = project_id
+    pl_p_status = ProjectStatus.LIVE.name
+    link_url = link
+    link_type = ProjectLinkType.OTHER.name
+
+    project_link = DbProjectLink(
+        id=project_link_id, project_id=pl_p_id, project_status=pl_p_status, link_url=link_url, link_type=link_type
+    )
+    print(project_link)
+    project_links.append(project_link)
+session.add_all(project_links)
+session.commit()
+
+# dataset artifact
+dataset_artifact_id = str(uuid.uuid4())
+da_d_id = dataset_id
+dataset_filename = filename
+dataset_filetype = DatasetArtifactFileType.H5AD.name
+dataset_type = DatasetArtifactType.REMIX.name
+user_submitted = True
+s3_uri = f"s3://corpora-api-dev/matrix/remix/{filename}"
```

Comment:
should this be parameterized for different stages?

Reply:
Sure - this script will not be re-used for other environments, but in case we recycle the code it would be good not to miss. Updating,